docs/sphinx/developer/arch/index.rst

```diff
@@ -5,7 +5,9 @@ SimEng architecture definitions are responsible for describing the features and
 
 To achieve this, SimEng defines a set of abstract architecture-related classes. Discrete implementations of these classes are provided for each of the ISAs SimEng supports by default, and must also be implemented for adding support for new or custom ISAs.
 
-Below provides more information on the abstract structure of a SimEng architecture and currently supported ISAs.
+ISA support is achieved through the use of the `Capstone <https://github.com/aquynh/capstone/>`_ disassembly framework, which disassembles binaries into C/C++ objects that include operand registers, access types, and immediate values to name a few. In order to update SimEng's AArch64 support from Armv8.4-a to Armv9.2-a, we undertook a Capstone update to allow for disassembly of the Armv9.2-a ISA. The work done for this can be found `here <https://github.com/capstone-engine/capstone/pull/1907>`_, and other useful ISA updating tools present in Capstone can be found `here <https://github.com/capstone-engine/capstone/tree/next/suite/synctools>`_.
```

Comment:
I think "which disassembles binaries into C/C++ objects that include" isn't exactly correct (as per comment https://github.com/UoB-HPC/SimEng/pull/254#discussion_r996291746). I will make a change to the wording in a new commit

Reply:
I have made this change